Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM navikt/java:17-appdynamics
+FROM ghcr.io/navikt/baseimages/temurin:19
```

Comment:
Inneholder denne appdynamics?

Reply:
Tror ikke det, men `temurin:19-appdynamics` skal vist gjøre det. Se: https://github.com/navikt/baseimages/blob/f4471d4c75481db2d403af7944c75df64a291624/.github/workflows/master.yml#L161

Reply:
Supert, endrer til det! :)

Reply:
@AudunSorheim Ikke sikkert det er støtte for AppD med Java 19 for temurin images lenger, må kanskje velge mellom 17 og 21: https://github.com/navikt/baseimages/blob/f4471d4c75481db2d403af7944c75df64a291624/.github/workflows/master.yml#L161

Reply:
Antagelig fordi 19 ikke er LTS